### PR TITLE
Load s2v model

### PR DESCRIPTION
**Pull request by louwenjjr**

### CODE REVIEW

File: ms2query_app.py [hunk 1]
```diff
@@ -80,3 +107,9 @@
 
 query_spectrums = [post_process_s2v(spec) for spec in query_spectrums]
 library_spectrums = [post_process_s2v(spec) for spec in library_spectrums]
+
+# turn spectra into documents
+documents_query = [SpectrumDocument(spec, n_decimals=2) for spec in
+                   query_spectrums]
+documents_library = [SpectrumDocument(spec, n_decimals=2) for spec in
+                     library_spectrums]
```
Comment (florian-huber) [inline ms2query_app.py:115]: That is the same as in #14 right? Just see my comment there on moving this part into an own function.

**Comment by florian-huber** [comment]
Nice progress Joris!
When running the app it worked fine when I entered a path, but it didn't when I drag/dropped or browsed the file.

**Review by florian-huber — CHANGES_REQUESTED** [review]
Nice additions!
Some extra changes would be nice, see comments.

**Comment by louwenjjr** [comment]
> Nice progress Joris!
> When running the app it worked fine when I entered a path, but it didn't when I drag/dropped or browsed the file.

My idea was to add the model file via the drag/drop box and then add the directory of the model file with entering a path. But with this comment I think it makes more sense for now to just enter the path to the model.

**Comment by florian-huber** [comment]
I agree, I think for now entering a path should be fine!
And it also makes me realize that we should at some point discuss the exact scope of a first prototype app. For instance, is it necessary to import an own library and model, or do we just provide defaults to try it out? But let's have those discussions elsewhere :)

File: ms2query_app.py [hunk 2]
```diff
@@ -58,7 +62,30 @@
...
 
-# processing of query and library spectra
+# write library info
+if library_spectrums:
+    st.write(f"Your library contains {len(library_spectrums)} spectra.")
+
+# load a s2v model in sidebar
+# todo: make more user friendly, currently there is no standard func to do this
+# for quick testing C:\Users\joris\Documents\eScience_data\data\trained_models\spec2vec_library_testing_4000removed_2dec.model
+model_file = st.sidebar.text_input("Enter path to Spec2Vec .model file:")
```
Comment (florian-huber) [inline ms2query_app.py:72]:
```suggestion
model_file = st.sidebar.text_input("Enter filename of Spec2Vec model (with path):")
```

**Review by florian-huber — APPROVED** [review]
Looking good!
Just had a minor comment on some text field.

**Pull request merged**